pilot/pkg/xds/ads_test.go

```diff
@@ -22,6 +23,10 @@ import (
 	discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
 	"github.com/golang/protobuf/proto"
 
+	"istio.io/istio/security/pkg/nodeagent/cache"
+
```

Comment:
remove blank line

Reply:
Strange - I "make format" and let the IDE deal with all imports, I was hoping we're past manual import editing... 
I'll fix it in the next commit.

Reply:
I run into this as well.. opened https://github.com/istio/istio/issues/25129 since this shouldn't something we care about in PRs